src/blog/2025-06-01-oxlint-stable.md

```diff
@@ -0,0 +1,158 @@
+For larger projects or projects that require more customization, Oxlint is configurable via an `.oxlintrc.json` file.
+This configuration format is based on ESLint v8’s flat config, making migration easy and familiar.
+Each source file is linted with the nearest applicable configuration, and you can use overrides to target specific glob patterns.
+You can also extend shared configs to keep teams consistent.
+
+For projects already using ESLint, [oxlint-migrate](https://github.com/oxc-project/oxlint-migrate) can be used to migrate an existing ESLint flat-config file to Oxlint. Additionally, eslint-plugin-oxlint can disable overlapping ESLint rules while both linters are used together.
+It is recommended to run `oxlint && eslint` to benefit from Oxlint's faster feedback cycle.
+
+For more detailed instructions on how to use Oxlint and integrate it with your project or editor, check out the [installation guide](/docs/guide/usage/linter).
+
+## Highlights
+
+### Comprehensive Rule Coverage
+
+Oxlint includes over 500 rules from various sources:
+
+- Complete ESLint rule set, including TypeScript-specific rules from typescript-eslint (Excluding type checked rules.)
+- Popular plugin rules from `eslint-plugin-unicorn`, `eslint-plugin-jsdoc`, `eslint-plugin-react`, `eslint-plugin-react-hooks`, `eslint-plugin-jest`, and `eslint-plugin-import`
+- Unique Oxlint rules like [bad comparison sequence](../docs/guide/usage/linter/rules/oxc/bad-comparison-sequence), [const comparisons](../docs/guide/usage/linter/rules/oxc/const-comparisons), and [only used in recursion](../docs/guide/usage/linter/rules/oxc/only-used-in-recursion)
+
+### Flexible Configuration
+
+Configure Oxlint through `.oxlintrc.json` files with support for:
+
+- Nested configurations that apply to specific directories
+- Override patterns for targeting specific file types or locations
+- Shared configuration extending for team consistency
+
+### Editor Integration
+
+First-class editor support with extensions for:
+
+- [VS Code](https://marketplace.visualstudio.com/items?itemName=oxc.oxc-vscode)
+- [IntelliJ IDEA and WebStorm](https://plugins.jetbrains.com/plugin/27061-oxc)
+- [Zed Editor](https://zed.dev/extensions?query=oxc)
+- Language server protocol support for other editors
+
+### Helpful Diagnostics
+
+Oxlint is built to deliver clear, actionable error messages - not just describing the issue, but visualizing it and suggesting how to fix it.
+
+![CLI Demo](https://github.com/oxc-project/oxc/assets/1430279/094a3b24-0433-42ae-aad2-48a7dec2b985)
+_Oxlint running in the terminal with detailed error reporting_
+
+## Benchmark
+
+Our [benchmark](https://github.com/oxc-project/bench-javascript-linter) reveals that Oxlint is around 50 - 100 times faster than ESLint with the same setup.
+
+| Tool                   | Time     |
+| ---------------------- | -------- |
+| oxlint (multi thread)  | 615.3 ms |
+| oxlint (single thread) | 1.840 s  |
+| eslint                 | 33.481 s |
+
+## Roadmap
```

Comment:
If we can, adding info around https://github.com/oxc-project/oxc/issues/9809 & https://github.com/oxc-project/oxc/issues/3105 could be helpful too